ui/react-ui/src/components/routes/userTasks/UserTasks.js

```diff
@@ -1,34 +1,48 @@
 import React, {useState, useEffect} from 'react';
 import Container from 'react-bootstrap/esm/Container';
-import {useParams} from "react-router-dom";
+import {useParams, useHistory} from "react-router-dom";
 import BootstrapTable from '../../bootstrapTable/BootstrapTable.js';
+import Button from 'react-bootstrap/Button';
+
+
 
 var UserTasks = () => {
-  const [userTasks, setUserTasks] = useState([]);
+  var [userTasks, setUserTasks] = useState(['loading']);
 
   var {userId} = useParams();
+  const history = useHistory();
 
   useEffect(() => {
-    // url should be replaced with => https://nsc-functionsapp-team1.azurewebsites.net/api/${userId}/tasks?
-    fetch(`https://w21httptriggernataliatest.azurewebsites.net/api/${userId}/tasks?`)
+    setUserTasks(['loading']);
+
+    fetch(`https://nsc-functionsapp-team1.azurewebsites.net/api/users/${userId}/tasks?`)
     .then(response => response.json())
     .then(data => setUserTasks(data))
     .catch((error) => console.error(error))
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  var tasks;
+  var backHomeButtonStyle = {marginLeft: '1rem', marginBottom: '2rem'};
+  var backIcon = '<====';
+
+  if (userTasks[0] === 'loading') tasks = <Container>...loading</Container>;
+  else {
+    if (userTasks.length > 0) {
+      tasks = <BootstrapTable heatherItems={Object.keys(userTasks[0])} rows={userTasks} />;
+    }
+    else tasks = <h5>There is no task for this user</h5>;
```

Comment:
I like this structure; it's much easier to read than the previous version. 
Did the old version work? Is the appearance (browser) the same as the new version?

Reply:
yes, it worked, just changed it to be more readable.